easybuild/easyblocks/__init__.py

```diff
@@ -32,9 +32,10 @@
 @author: Jens Timmerman (Ghent University)
 """
 import os
-from distutils.version import LooseVersion
 from pkgutil import extend_path
 
+from easybuild.tools.loose_version import LooseVersion
```

Comment:
I think it's specifically this import that trips up the CI, this is the thing that's used for installed the easyblocks pip package itself.

Really not sure how we fix this though...